sisl/io/siesta/ani.py

```diff
@@ -0,0 +1,61 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+"""
+Sile object for reading ANI files
+"""
+import numpy as np
+
+from .sile import SileSiesta
+from ..sile import add_sile, sile_fh_open
+from sisl._internal import set_module
+from sisl import Geometry, SuperCell
+
+__all__ = ["aniSileSiesta"]
+
+@set_module("sisl.io.siesta")
+class aniSileSiesta(SileSiesta):
+    """ ANI file object """
+
+    @sile_fh_open(True)
+    def read_geometry(self, step=1):
+        """ Reads the geometries from SIESTA ANI file
+
+        Parameters
+        ----------
+        step: int, optional
+            step size for geometries to be extracted
+
+        Returns
+        -------
+        geometries: list of Geometry
+        """
+
+        def readG(itt):
+            atoms = []
+            xyz = []
+            # Read the Geometry block
+            try:
+                na = int(next(itt))
+            except:
+                return None
+            next(itt) # blank line
```

Comment:
The ANI file format sometimes allows the blank line to contain a single number, I know that molden can use this to plot, say the total energy. Would it make sense to return this upon request?

Reply:
Yes, good point.